app/celery/scheduled_tasks.py

```diff
@@ -253,3 +256,42 @@ def check_for_missing_rows_in_completed_jobs():
                     current_app.logger.info(
                         "Processing missing row: {} for job: {}".format(row_to_process.missing_row, job.id))
                     process_row(row, template, job, job.service, sender_id=sender_id)
+
+
+@notify_celery.task(name='check-for-services-with-high-failure-rates-or-sending-to-tv-numbers')
+@statsd(namespace="tasks")
+def check_for_services_with_high_failure_rates_or_sending_to_tv_numbers():
+    start_date = (datetime.utcnow() - timedelta(days=1))
+    end_date = datetime.utcnow()
+    message = ""
+
+    services_with_failures = get_services_with_high_failure_rates(start_date=start_date, end_date=end_date)
+    services_sending_to_tv_numbers = dao_find_services_sending_to_tv_numbers(start_date=start_date, end_date=end_date)
+
+    if services_with_failures:
+        message += "{} service(s) have had high permanent-failure rates for sms messages in last 24 hours:\n".format(
+            len(services_with_failures)
+        )
+        for service in services_with_failures:
+            message += "service id: {} failure rate: {},\n".format(service["id"], service["permanent_failure_rate"])
+    elif services_sending_to_tv_numbers:
+        message += "{} service(s) have sent over 100 sms messages to tv numbers in last 24 hours:\n".format(
+            len(services_sending_to_tv_numbers)
+        )
+        for service in services_sending_to_tv_numbers:
+            message += "service id: {}, count of sms to tv numbers: {},\n".format(
+                service.service_id, service.notification_count
+            )
+
+    if services_with_failures or services_sending_to_tv_numbers:
+        current_app.logger.exception(message)
+
+        if current_app.config['NOTIFY_ENVIRONMENT'] in ['live', 'production', 'test']:
+            message += "\nThings to do: contact service? revoke their key?"
```

Comment:
Explain what TV numbers are:
These are numbers reserved for the use in TV and drama. The range of TV numbers are 07700 900000 to 900999. When an SMS notification is sent to such a number, we get a 'permanent-failure' status back and we are billed for this notification.

Reply:
Thanks. That first sentence would be useful to put in the TODO section or in the runbook so a developer receiving the ticket knows how to deal with it

Reply:
yeah, I wrote it to myself here only to remember about including it :). As it only transpired in a spoken conversation.